resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
+                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "type": "bool",
```

Comment:
@BagelOrb I discussed with Ruben on this one. I thought it is not very clear what skip count = 0 means, so I made this boolean.